ts-client/src/dlmm/index.ts

```diff
@@ -929,6 +930,98 @@ export class DLMM {
     ]);
   }
 
+  /**
+   * The function `getBinArrayAroundActiveBin` retrieves a specified number of `BinArrayAccount`
+   * objects from the blockchain, based on the active bin and its surrounding bin arrays.
+   * @param [count=2] - The `count` parameter is the number of bin arrays to retrieve on left and right respectively. By default, it
+   * is set to 2.
+   * @returns an array of `BinArrayAccount` objects.
+   */
+  public async getBinArrayAroundActiveBin(
```

Comment:
I think in swap we should use function with input as swapForY: boolean
we dont need to fetch 2 sides for binArrays
Ex: in jupiter launch, we only have a side for buy at the begining